? UMP-Albania/src/POI-Albania.inne.pnt-converted.txt
? UMP-Albania/src/cities-Albania.pnt-converted.txt
? UMP-Andora/src/POI-Andora.inne.pnt-converted.txt
? UMP-Andora/src/cities-Andora.pnt-converted.txt
? UMP-Austria/src/AUSTRIA.debug.pnt-converted.txt
? UMP-Austria/src/POI-Austria.inne.pnt-converted.txt
? UMP-Austria/src/POI-Austria.paliwo.pnt-converted.txt
? UMP-Austria/src/POI-ROBOCZY.pnt-converted.txt
? UMP-Austria/src/cities-Austria.pnt-converted.txt
? UMP-Belgia/src/BELGIA.debug.pnt-converted.txt
? UMP-Belgia/src/POI-Belgia.inne.pnt-converted.txt
? UMP-Belgia/src/POI-Belgia.nawigacja.pnt-converted.txt
? UMP-Belgia/src/POI-Belgia.paliwo.pnt-converted.txt
? UMP-Belgia/src/cities-Belgia.pnt-converted.txt
? UMP-Bialorus/src/POI-Bialorus.inne.pnt-converted.txt
? UMP-Bialorus/src/cities-Bialorus.pnt-converted.txt
? UMP-Bialystok/src/BIALYSTOK.debug.pnt-converted.txt
? UMP-Bialystok/src/BIALYSTOK.routing.pnt-converted.txt
? UMP-Bialystok/src/POI-Bialystok.inne.pnt-converted.txt
? UMP-Bialystok/src/POI-Bialystok.koscioly.pnt-converted.txt
? UMP-Bialystok/src/POI-Bialystok.paliwo.pnt-converted.txt
? UMP-Bialystok/src/POI-Bialystok.topo.pnt-converted.txt
? UMP-Bialystok/src/cities-Bialystok.pnt-converted.txt
? UMP-Bosnia/src/BOSNIA_HERCEGOWINA.debug.pnt-converted.txt
? UMP-Bosnia/src/POI-Bosnia_Hercegowina.inne.pnt-converted.txt
? UMP-Bosnia/src/cities-BosniaHercegowina.pnt-converted.txt
? UMP-Bulgaria/src/BULGARIA.debug.pnt-converted.txt
? UMP-Bulgaria/src/POI-Bulgaria.inne.pnt-converted.txt
? UMP-Bulgaria/src/POI-Bulgaria.nawigacja.pnt-converted.txt
? UMP-Bulgaria/src/cities-Bulgaria.pnt-converted.txt
? UMP-Chorwacja/src/CHORWACJA.debug.pnt-converted.txt
? UMP-Chorwacja/src/POI-Chorwacja.geografia.pnt-converted.txt
? UMP-Chorwacja/src/POI-Chorwacja.inne.pnt-converted.txt
? UMP-Chorwacja/src/POI-Chorwacja.nawigacja.pnt-converted.txt
? UMP-Chorwacja/src/POI-Chorwacja.paliwo.pnt-converted.txt
? UMP-Chorwacja/src/cities-Chorwacja.pnt-converted.txt
? UMP-Ciechanow/src/CIECHANOW.debug.pnt-converted.txt
? UMP-Ciechanow/src/CIECHANOW.szlaki.topo.pnt-converted.txt
? UMP-Ciechanow/src/POI-Ciechanow.bankomaty.pnt-converted.txt
? UMP-Ciechanow/src/POI-Ciechanow.inne.pnt-converted.txt
? UMP-Ciechanow/src/POI-Ciechanow.jedzenie.pnt-converted.txt
? UMP-Ciechanow/src/POI-Ciechanow.kolej.pnt-converted.txt
? UMP-Ciechanow/src/POI-Ciechanow.paliwo.pnt-converted.txt
? UMP-Ciechanow/src/cities-Ciechanow.pnt-converted.txt
? UMP-Cypr/src/CYPR.debug.pnt-converted.txt
? UMP-Cypr/src/POI-Cypr.inne.pnt-converted.txt
? UMP-Cypr/src/POI-Cypr.jedzenie.pnt-converted.txt
? UMP-Cypr/src/POI-Cypr.koscioly.pnt-converted.txt
? UMP-Cypr/src/POI-Cypr.nawigacja.pnt-converted.txt
? UMP-Cypr/src/POI-Cypr.paliwo.pnt-converted.txt
? UMP-Cypr/src/cities-Cypr.pnt-converted.txt
? UMP-Czarnogora/src/CZARNOGORA.debug.pnt-converted.txt
? UMP-Czarnogora/src/POI-Czarnogora.inne.pnt-converted.txt
? UMP-Czarnogora/src/cities-Czarnogora.pnt-converted.txt
? UMP-Czechy/src/CZECHY.debug.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.geografia.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.hotele.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.inne.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.jedzenie.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.kolej.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.paliwo.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.rozrywka.pnt-converted.txt
? UMP-Czechy/src/POI-Czechy.sklepy.pnt-converted.txt
? UMP-Czechy/src/cities-Czechy.pnt-converted.txt
? UMP-Dania/src/DANIA.debug.pnt-converted.txt
? UMP-Dania/src/POI-Dania.geografia.pnt-converted.txt
? UMP-Dania/src/POI-Dania.inne.pnt-converted.txt
? UMP-Dania/src/POI-Dania.nawigacja.pnt-converted.txt
? UMP-Dania/src/POI-Dania.paliwo.pnt-converted.txt
? UMP-Dania/src/cities-Dania.pnt-converted.txt
? UMP-Estonia/src/POI-Estonia.inne.pnt-converted.txt
? UMP-Estonia/src/POI-Estonia.nawigacja.pnt-converted.txt
? UMP-Estonia/src/cities-Estonia.pnt-converted.txt
? UMP-Finlandia/src/POI-Finlandia.Nawigacja.pnt-converted.txt
? UMP-Finlandia/src/POI-Finlandia.inne.pnt-converted.txt
? UMP-Finlandia/src/cities-Finlandia.pnt-converted.txt
? UMP-Francja/src/FRANCJA.debug.pnt-converted.txt
? UMP-Francja/src/POI-Corse.inne.pnt-converted.txt
? UMP-Francja/src/POI-Corse.nawigacja.pnt-converted.txt
? UMP-Francja/src/POI-Corse.paliwo.pnt-converted.txt
? UMP-Francja/src/POI-Corse.zabytki.pnt-converted.txt
? UMP-Francja/src/POI-Francja.inne.pnt-converted.txt
? UMP-Francja/src/POI-Francja.nawigacja.pnt-converted.txt
? UMP-Francja/src/POI-Francja.paliwo.pnt-converted.txt
? UMP-Francja/src/POI-Francja.zabytki.pnt-converted.txt
? UMP-Francja/src/cities-Corse.pnt-converted.txt
? UMP-Francja/src/cities-Francja.pnt-converted.txt
? UMP-Gdansk/src/POI-Baltyk.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.bankomaty.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.budynki.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.geografia.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.hotele.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.inne.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.jedzenie.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.kolej.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.koscioly.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.mosty.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.muzea.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.paliwo.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.routing.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.sklepy.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.szkoly.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.szlaki.topo.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.szpitale.pnt-converted.txt
? UMP-Gdansk/src/POI-GDANSK.topo.pnt-converted.txt
? UMP-Gdansk/src/cities-Gdansk.pnt-converted.txt
? UMP-GorzowWlkp/src/GORZOW.debug.pnt-converted.txt
? UMP-GorzowWlkp/src/POI-Gorzow.bankomaty.pnt-converted.txt
? UMP-GorzowWlkp/src/POI-Gorzow.inne.pnt-converted.txt
? UMP-GorzowWlkp/src/POI-Gorzow.paliwo.pnt-converted.txt
? UMP-GorzowWlkp/src/POI-Gorzow.topo.pnt-converted.txt
? UMP-GorzowWlkp/src/cities-Gorzow.pnt-converted.txt
? UMP-GorzowWlkp/src/inne.zakazy.pnt-converted.txt
? UMP-Grecja/src/GRECJA.debug.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.geografia.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.inne.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.jedzenie.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.nawigacja.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.paliwo.pnt-converted.txt
? UMP-Grecja/src/POI-Grecja.zabytki.pnt-converted.txt
? UMP-Grecja/src/cities-Grecja.pnt-converted.txt
? UMP-Hiszpania/src/HISZPANIA.debug.pnt-converted.txt
? UMP-Hiszpania/src/Hiszpania-miasta_poza_obszarem_mapy.pnt.txt-converted.txt
? UMP-Hiszpania/src/POI-Hiszpania.inne.pnt-converted.txt
? UMP-Hiszpania/src/POI-Hiszpania.nawigacja.pnt-converted.txt
? UMP-Hiszpania/src/POI-Hiszpania.paliwo.pnt-converted.txt
? UMP-Hiszpania/src/cities-Hiszpania.pnt-converted.txt
? UMP-Holandia/src/HOLANDIA.debug.pnt-converted.txt
? UMP-Holandia/src/POI-Holandia.inne.pnt-converted.txt
? UMP-Holandia/src/POI-Holandia.nawigacja.pnt-converted.txt
? UMP-Holandia/src/POI-Holandia.paliwo.pnt-converted.txt
? UMP-Holandia/src/cities-Holandia.pnt-converted.txt
? UMP-Irlandia/src/POI-Irlandia.inne.pnt-converted.txt
? UMP-Irlandia/src/POI-Irlandia.nawigacja.pnt-converted.txt
? UMP-Irlandia/src/POI-Irlandia.zabytki.pnt-converted.txt
? UMP-Irlandia/src/cities-Irlandia.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.bankomaty.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.benzyna.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.debug.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.routing.pnt-converted.txt
? UMP-JeleniaGora/src/POI-JeleniaGora.topo.pnt-converted.txt
? UMP-JeleniaGora/src/cities-JeleniaGora.pnt-converted.txt
? UMP-Kaliningrad/src/POI-Kaliningrad.nawigacja.pnt-converted.txt
? UMP-Kaliningrad/src/cities-Kaliningrad.pnt-converted.txt
? UMP-Kalisz/src/KALISZ.POI.bunkry.pnt-converted.txt
? UMP-Kalisz/src/KALISZ.topo.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.bankomaty.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.jedzenie.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.kolej.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.koscioly.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.paliwo.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.routing.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.sklepy.pnt-converted.txt
? UMP-Kalisz/src/POI-Kalisz.szpitale.pnt-converted.txt
? UMP-Kalisz/src/cities-Kalisz.pnt-converted.txt
? UMP-Katowice/src/POI-Cieszyn.pnt-converted.txt
? UMP-Katowice/src/POI-Katowice.pnt-converted.txt
? UMP-Katowice/src/POI-Ustron.pnt-converted.txt
? UMP-Katowice/src/POI-Wisla.pnt-converted.txt
? UMP-Katowice/src/POI-bankomaty.pnt-converted.txt
? UMP-Katowice/src/POI-jedzenie.pnt-converted.txt
? UMP-Katowice/src/POI-koscioly.pnt-converted.txt
? UMP-Katowice/src/POI-paliwo.pnt-converted.txt
? UMP-Katowice/src/POI-routing.pnt-converted.txt
? UMP-Katowice/src/POI-szpitale.pnt-converted.txt
? UMP-Katowice/src/cities-Katowice.pnt-converted.txt
? UMP-Kielce/src/KIELCE.debug.pnt-converted.txt
? UMP-Kielce/src/POI-Kielce.geografia.pnt-converted.txt
? UMP-Kielce/src/POI-Kielce.inne.pnt-converted.txt
? UMP-Kielce/src/POI-Kielce.koscioly.pnt-converted.txt
? UMP-Kielce/src/POI-Kielce.paliwo.pnt-converted.txt
? UMP-Kielce/src/cities-Kielce.pnt-converted.txt
? UMP-Klodzko/src/KLODZKO.debug.pnt-converted.txt
? UMP-Klodzko/src/POI-Glucholazy.pnt-converted.txt
? UMP-Klodzko/src/POI-Klodzko.pnt-converted.txt
? UMP-Klodzko/src/POI-Nysa.pnt-converted.txt
? UMP-Klodzko/src/POI-Paczkow.pnt-converted.txt
? UMP-Klodzko/src/cities-Klodzko.pnt-converted.txt
? UMP-Kosowo/src/POI-Kosowo.inne.pnt-converted.txt
? UMP-Kosowo/src/cities-Kosowo.pnt-converted.txt
? UMP-Koszalin/src/KOSZALIN.debug.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.budynki.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.geografia.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.hotele.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.inne.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.jedzenie.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.kolej.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.koscioly.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.mosty.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.paliwo.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.szkoly.pnt-converted.txt
? UMP-Koszalin/src/POI-KOSZALIN.topo.pnt-converted.txt
? UMP-Koszalin/src/cities-Koszalin.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.atrakcje.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.bankomaty.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.budynki.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.dorozlozenia.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.gastronomia.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.hotele.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.inne.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.koscioly.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.paliwo.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.rozrywka.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.szkoly.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.szpitale.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.transport.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.zabytki.pnt-converted.txt
? UMP-Krakow/src/KRAKOW.zakupy.pnt-converted.txt
? UMP-Krakow/src/POI-Krakow.inne.pnt-converted.txt
? UMP-Krakow/src/POI-routing.pnt-converted.txt
? UMP-Krakow/src/cities-Krakow.pnt-converted.txt
? UMP-Leszno/src/POI-Leszno.topo.pnt-converted.txt
? UMP-Leszno/src/POI-benzyna.pnt-converted.txt
? UMP-Leszno/src/POI-inne.pnt-converted.txt
? UMP-Leszno/src/POI-routing.pnt-converted.txt
? UMP-Leszno/src/cities-Leszno.pnt-converted.txt
? UMP-Lichtenstein/src/POI-Lichtenstein.inne.pnt-converted.txt
? UMP-Lichtenstein/src/cities-Lichtenstein.pnt-converted.txt
? UMP-Litwa/src/POI-Litwa.inne.pnt-converted.txt
? UMP-Litwa/src/cities-Litwa.pnt-converted.txt
? UMP-Lodz/src/LODZ.debug.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.atrakcje.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.bankomaty.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.bunkry.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.danger.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.hotele.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.jedzenie.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.koscioly.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.mosty.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.paliwa.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.parkingi.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.routing.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.sklepy.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.sport.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.szkoly.pnt-converted.txt
? UMP-Lodz/src/POI-BELCHATOW.zdrowie.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.bankomaty.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.benzyna.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.budynki.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.hotele.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.inne.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.jedzenie.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.rozrywka.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.sklepy.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.szpitale.pnt-converted.txt
? UMP-Lodz/src/POI-Lodz.topo.pnt-converted.txt
? UMP-Lodz/src/POI-routing.pnt-converted.txt
? UMP-Lodz/src/cities-Belchatow.pnt-converted.txt
? UMP-Lodz/src/cities-Lodz.pnt-converted.txt
? UMP-Lotwa/src/POI-Lotwa.inne.pnt-converted.txt
? UMP-Lotwa/src/cities-Lotwa.pnt-converted.txt
? UMP-Lublin/src/LUBLIN.kolej.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.bankomaty.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.hotele.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.jedzenie.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.koscioly.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.paliwo.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.sklepy.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.szkoly.pnt-converted.txt
? UMP-Lublin/src/POI-Lublin.szpitale.pnt-converted.txt
? UMP-Lublin/src/POI-routing.pnt-converted.txt
? UMP-Lublin/src/cities-Lublin.pnt-converted.txt
? UMP-Luksemburg/src/LUKSEMBURG.debug.pnt-converted.txt
? UMP-Luksemburg/src/POI-Luksemburg.inne.pnt-converted.txt
? UMP-Luksemburg/src/POI-Luksemburg.paliwo.pnt-converted.txt
? UMP-Luksemburg/src/cities-Luksemburg.pnt-converted.txt
? UMP-Macedonia/src/POI-Macedonia.inne.pnt-converted.txt
? UMP-Macedonia/src/cities-Macedonia.pnt-converted.txt
? UMP-Malta/src/POI-Malta.inne.pnt-converted.txt
? UMP-Malta/src/POI-Malta.nawigacja.pnt-converted.txt
? UMP-Malta/src/cities-Malta.pnt-converted.txt
? UMP-Moldawia/src/POI-Moldawia.inne.pnt-converted.txt
? UMP-Moldawia/src/cities-Moldawia.pnt-converted.txt
? UMP-Norwegia/src/POI-Norwegia.inne.pnt-converted.txt
? UMP-Norwegia/src/POI-Norwegia.nawigacja.pnt-converted.txt
? UMP-Norwegia/src/cities-Norwegia.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.bankomaty.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.debug.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.geografia.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.hotele.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.inne.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.jedzenie.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.koscioly.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.paliwo.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.rozrywka.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.sklepy.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.szkoly.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.szpitale.pnt-converted.txt
? UMP-NowySacz/src/NOWY_SACZ.zabytki.pnt-converted.txt
? UMP-NowySacz/src/POI-Limanowa.pnt-converted.txt
? UMP-NowySacz/src/POI-NowySacz.pnt-converted.txt
? UMP-NowySacz/src/POI-NowySacz.topo.pnt-converted.txt
? UMP-NowySacz/src/POI-NowySacz1.pnt-converted.txt
? UMP-NowySacz/src/POI-Zakopane.pnt-converted.txt
? UMP-NowySacz/src/ZAKOPANE.jedzenie.pnt-converted.txt
? UMP-NowySacz/src/cities-NowySacz.pnt-converted.txt
? UMP-Olsztyn/src/OLSZTYN.debug.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.geografia.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.hotele.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.inne.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.jedzenie.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.koscioly.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.mosty.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.paliwo.pnt-converted.txt
? UMP-Olsztyn/src/POI-Olsztyn.topo.pnt-converted.txt
? UMP-Olsztyn/src/cities-Olsztyn.pnt-converted.txt
? UMP-Opole/src/OPOLE.routing.pnt-converted.txt
? UMP-Opole/src/OPOLE.topo.pnt-converted.txt
? UMP-Opole/src/POI-Olesno.pnt-converted.txt
? UMP-Opole/src/POI-Opole.paliwo.pnt-converted.txt
? UMP-Opole/src/POI-Opole.pnt-converted.txt
? UMP-Opole/src/cities-Opole.pnt-converted.txt
? UMP-Pila/src/POI-PILA.routing.pnt-converted.txt
? UMP-Pila/src/POI-Pila.budynki.pnt-converted.txt
? UMP-Pila/src/POI-Pila.inne.pnt-converted.txt
? UMP-Pila/src/POI-Pila.koscioly.pnt-converted.txt
? UMP-Pila/src/POI-Pila.kultura.pnt-converted.txt
? UMP-Pila/src/POI-Pila.mosty.pnt-converted.txt
? UMP-Pila/src/POI-Pila.paliwo.pnt-converted.txt
? UMP-Pila/src/POI-Pila.topo.pnt-converted.txt
? UMP-Pila/src/cities-Pila.pnt-converted.txt
? UMP-Plock/src/PLOCK.bankomaty.pnt-converted.txt
? UMP-Plock/src/PLOCK.budynki.pnt-converted.txt
? UMP-Plock/src/PLOCK.debug.topo.pnt-converted.txt
? UMP-Plock/src/PLOCK.hotele.pnt-converted.txt
? UMP-Plock/src/PLOCK.inne.pnt-converted.txt
? UMP-Plock/src/PLOCK.jedzenie.pnt-converted.txt
? UMP-Plock/src/PLOCK.kolej.pnt-converted.txt
? UMP-Plock/src/PLOCK.koscioly.pnt-converted.txt
? UMP-Plock/src/PLOCK.paliwo.pnt-converted.txt
? UMP-Plock/src/PLOCK.routing.pnt-converted.txt
? UMP-Plock/src/PLOCK.rozrywka.pnt-converted.txt
? UMP-Plock/src/PLOCK.sklepy.pnt-converted.txt
? UMP-Plock/src/PLOCK.sport.pnt-converted.txt
? UMP-Plock/src/PLOCK.szkoly.pnt-converted.txt
? UMP-Plock/src/PLOCK.szpitale.pnt-converted.txt
? UMP-Plock/src/PLOCK.topo.pnt-converted.txt
? UMP-Plock/src/PLOCK.zabytki.pnt-converted.txt
? UMP-Plock/src/cities-Plock.pnt-converted.txt
? UMP-Portugalia/src/POI-Portugalia.inne.pnt-converted.txt
? UMP-Portugalia/src/POI-Portugalia.nawigacja.pnt-converted.txt
? UMP-Portugalia/src/Portugalia-miasta_poza_obszarem_mapy.pnt.txt-converted.txt
? UMP-Portugalia/src/cities-Portugalia.pnt-converted.txt
? UMP-Poznan/src/POI-Poznan.edu.przedszkola.pnt-converted.txt
? UMP-Poznan/src/POI-Poznan.historia.pnt-converted.txt
? UMP-Poznan/src/POI-Poznan.szlaki.topo.pnt-converted.txt
? UMP-Poznan/src/POI-Poznan.topo.pnt-converted.txt
? UMP-Poznan/src/POI-banki.pnt-converted.txt
? UMP-Poznan/src/POI-hotele.pnt-converted.txt
? UMP-Poznan/src/POI-inne.pnt-converted.txt
? UMP-Poznan/src/POI-jedzenie.pnt-converted.txt
? UMP-Poznan/src/POI-paliwo.pnt-converted.txt
? UMP-Poznan/src/cities-Poznan.pnt-converted.txt
? UMP-Poznan/src/inne.zakazy.pnt-converted.txt
? UMP-Przemysl/src/POI-Przemysl.pnt-converted.txt
? UMP-Przemysl/src/POI-routing.pnt-converted.txt
? UMP-Przemysl/src/cities-Przemysl.pnt-converted.txt
? UMP-Radom/src/POI-Radom.bankomaty.pnt-converted.txt
? UMP-Radom/src/POI-Radom.paliwo.pnt-converted.txt
? UMP-Radom/src/POI-Radom.pnt-converted.txt
? UMP-Radom/src/RADOM.debug.pnt-converted.txt
? UMP-Radom/src/cities-Radom.pnt-converted.txt
? UMP-Rosja/src/POI-Rosja.inne.pnt-converted.txt
? UMP-Rosja/src/POI-Rosja.nawigacja.pnt-converted.txt
? UMP-Rosja/src/cities-Rosja.pnt-converted.txt
? UMP-Rumunia/src/POI-Rumunia.inne.pnt-converted.txt
? UMP-Rumunia/src/POI-Rumunia.nawigacja.pnt-converted.txt
? UMP-Rumunia/src/cities-Rumunia.pnt-converted.txt
? UMP-Rzeszow/src/POI-Debica.pnt-converted.txt
? UMP-Rzeszow/src/POI-Rzeszow.paliwo.pnt-converted.txt
? UMP-Rzeszow/src/POI-Rzeszow.pnt-converted.txt
? UMP-Rzeszow/src/POI-Rzeszow.szkoly.pnt-converted.txt
? UMP-Rzeszow/src/POI-routing.pnt-converted.txt
? UMP-Rzeszow/src/cities-Rzeszow.pnt-converted.txt
? UMP-Serbia/src/POI-Serbia.inne.pnt-converted.txt
? UMP-Serbia/src/SERBIA.debug.pnt-converted.txt
? UMP-Serbia/src/cities-Serbia.pnt-converted.txt
? UMP-Siedlce/src/POI-Siedlce.bankomaty.pnt-converted.txt
? UMP-Siedlce/src/POI-Siedlce.paliwo.pnt-converted.txt
? UMP-Siedlce/src/POI-Siedlce.pnt-converted.txt
? UMP-Siedlce/src/SIEDLCE.debug.pnt-converted.txt
? UMP-Siedlce/src/SIEDLCE.topo.pnt-converted.txt
? UMP-Siedlce/src/cities-Siedlce.pnt-converted.txt
? UMP-Slowacja/src/POI-Slowacja.geografia.pnt-converted.txt
? UMP-Slowacja/src/POI-Slowacja.inne.pnt-converted.txt
? UMP-Slowacja/src/POI-Slowacja.paliwo.pnt-converted.txt
? UMP-Slowacja/src/SLOWACJA.debug.pnt-converted.txt
? UMP-Slowacja/src/cities-Slowacja.pnt-converted.txt
? UMP-Slowenia/src/POI-Slowenia.inne.pnt-converted.txt
? UMP-Slowenia/src/SLOWENIA.debug.pnt-converted.txt
? UMP-Slowenia/src/cities-Slowenia.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.geografia.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.inne.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.mosty.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.paliwo.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.sport.pnt-converted.txt
? UMP-Suwalki/src/POI-Suwalki.topo.pnt-converted.txt
? UMP-Suwalki/src/SUWALKI.debug.pnt-converted.txt
? UMP-Suwalki/src/SUWALKI.szlaki.topo.pnt-converted.txt
? UMP-Suwalki/src/cities-Suwalki.pnt-converted.txt
? UMP-Szczecin/src/POI-Baltyk.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.bankomaty.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.budynki.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.geografia.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.hotele.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.inne.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.jedzenie.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.kolej.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.koscioly.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.mosty.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.paliwo.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.sklepy.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.szkoly.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.szpitale.pnt-converted.txt
? UMP-Szczecin/src/POI-Szczecin.topo.pnt-converted.txt
? UMP-Szczecin/src/SZCZECIN.debug.pnt-converted.txt
? UMP-Szczecin/src/cities-Szczecin.pnt-converted.txt
? UMP-Szwajcaria/src/POI-Szwajcaria.inne.pnt-converted.txt
? UMP-Szwajcaria/src/SZWAJCARIA.debug.pnt-converted.txt
? UMP-Szwajcaria/src/cities-Szwajcaria.pnt-converted.txt
? UMP-Szwecja/src/POI-Szwecja.debug.pnt-converted.txt
? UMP-Szwecja/src/POI-Szwecja.geografia.pnt-converted.txt
? UMP-Szwecja/src/POI-Szwecja.inne.pnt-converted.txt
? UMP-Szwecja/src/POI-Szwecja.nawigacja.pnt-converted.txt
? UMP-Szwecja/src/POI-Szwecja.porty.pnt-converted.txt
? UMP-Szwecja/src/cities-Szwecja.pnt-converted.txt
? UMP-Tarnow/src/POI-bankomaty.pnt-converted.txt
? UMP-Tarnow/src/POI-debug.pnt-converted.txt
? UMP-Tarnow/src/POI-inne.pnt-converted.txt
? UMP-Tarnow/src/POI-koscioly.pnt-converted.txt
? UMP-Tarnow/src/POI-paliwo.pnt-converted.txt
? UMP-Tarnow/src/POI-szkoly.pnt-converted.txt
? UMP-Tarnow/src/POI-szpitale.pnt-converted.txt
? UMP-Tarnow/src/cities-Tarnow.pnt-converted.txt
? UMP-Torun/src/BYDGOSZCZ.punkty.pnt-converted.txt
? UMP-Torun/src/BYDGOSZCZ.routing.pnt-converted.txt
? UMP-Torun/src/DROGI.routing.pnt-converted.txt
? UMP-Torun/src/POI-Torun.pnt-converted.txt
? UMP-Torun/src/POI-routing.pnt-converted.txt
? UMP-Torun/src/cities-Torun.pnt-converted.txt
? UMP-Turcja/src/POI-Turcja.inne.pnt-converted.txt
? UMP-Turcja/src/POI-Turcja.nawigacja.pnt-converted.txt
? UMP-Turcja/src/TURCJA.debug.pnt-converted.txt
? UMP-Turcja/src/cities-Turcja.pnt-converted.txt
? UMP-Ukraina/src/POI-Ukraina.geografia.pnt-converted.txt
? UMP-Ukraina/src/POI-Ukraina.inne.pnt-converted.txt
? UMP-Ukraina/src/UKRAINA.debug.pnt-converted.txt
? UMP-Ukraina/src/cities-Ukraina.pnt-converted.txt
? UMP-Ukraina/src/miasta_poza_obszarem_mapy.pnt.txt-converted.txt
? UMP-Warszawa/src/WARSZAWA.bankomaty.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.budynki.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.debug.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.debug.topo.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.historia.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.hotele.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.inne.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.jedzenie.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.kinoteatr.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.kolej.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.koscioly.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.msi.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.paliwo.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.rozrywka.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.sklepy.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.sport.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.szkoly.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.szlaki.topo.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.szpitale.pnt-converted.txt
? UMP-Warszawa/src/WARSZAWA.zabytki.pnt-converted.txt
? UMP-Warszawa/src/cities-Warszawa.pnt-converted.txt
? UMP-Wegry/src/POI-Wegry.inne.pnt-converted.txt
? UMP-Wegry/src/WEGRY.debug.pnt-converted.txt
? UMP-Wegry/src/cities-Wegry.pnt-converted.txt
? UMP-WielkaBrytania/src/POI-Anglia.inne.pnt-converted.txt
? UMP-WielkaBrytania/src/POI-IrlandiaPln.inne.pnt-converted.txt
? UMP-WielkaBrytania/src/POI-Owcze.pnt-converted.txt
? UMP-WielkaBrytania/src/POI-Szkocja.inne.pnt-converted.txt
? UMP-WielkaBrytania/src/POI-WielkaBrytania.nawigacja.pnt-converted.txt
? UMP-WielkaBrytania/src/WIELKA_BRYTANIA.debug.pnt-converted.txt
? UMP-WielkaBrytania/src/cities-IrlandiaPln.pnt-converted.txt
? UMP-WielkaBrytania/src/cities-Owcze.pnt-converted.txt
? UMP-WielkaBrytania/src/cities-WielkaBrytania.pnt-converted.txt
? UMP-Wlochy/src/EMILIAROMANIA.debug.pnt-converted.txt
? UMP-Wlochy/src/LOMBARDIA.debug.pnt-converted.txt
? UMP-Wlochy/src/LUGRIA.debug.pnt-converted.txt
? UMP-Wlochy/src/PIEMONTE.debug.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Abruzzo.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Basilicata.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Calabria.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Calabria.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Calabria.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Calabria.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Campania.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-EmiliaRomania.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Lazio.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Lombardia.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.bankomaty.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Lugria.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Piemonte.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Sardegna.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Sicilia.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Toscana.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Trentino.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-UmbriaMarche.zabytki.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.inne.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.jedzenie.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.koscioly.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.nawigacja.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.paliwo.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.sklepy.pnt-converted.txt
? UMP-Wlochy/src/POI-Veneto.zabytki.pnt-converted.txt
? UMP-Wlochy/src/TOSCANA.debug.pnt-converted.txt
? UMP-Wlochy/src/TRENTINO.debug.pnt-converted.txt
? UMP-Wlochy/src/VENETO.debug.pnt-converted.txt
? UMP-Wlochy/src/cities-Abruzzo.pnt-converted.txt
? UMP-Wlochy/src/cities-Basilicata.pnt-converted.txt
? UMP-Wlochy/src/cities-Calabria.pnt-converted.txt
? UMP-Wlochy/src/cities-Campania.pnt-converted.txt
? UMP-Wlochy/src/cities-EmiliaRomania.pnt-converted.txt
? UMP-Wlochy/src/cities-Lazio.pnt-converted.txt
? UMP-Wlochy/src/cities-Lombardia.pnt-converted.txt
? UMP-Wlochy/src/cities-Lugria.pnt-converted.txt
? UMP-Wlochy/src/cities-Piemonte.pnt-converted.txt
? UMP-Wlochy/src/cities-Sardegna.pnt-converted.txt
? UMP-Wlochy/src/cities-Sicilia.pnt-converted.txt
? UMP-Wlochy/src/cities-Toscana.pnt-converted.txt
? UMP-Wlochy/src/cities-Trentino.pnt-converted.txt
? UMP-Wlochy/src/cities-UmbriaMarche.pnt-converted.txt
? UMP-Wlochy/src/cities-Veneto.pnt-converted.txt
? UMP-Wloclawek/src/POI-Wloclawek.pnt-converted.txt
? UMP-Wloclawek/src/POI-benzyna.pnt-converted.txt
? UMP-Wloclawek/src/POI-routing.pnt-converted.txt
? UMP-Wloclawek/src/cities-Wloclawek.pnt-converted.txt
? UMP-Wroclaw/src/POI-Legnica-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Legnica.pnt-converted.txt
? UMP-Wroclaw/src/POI-Lubin-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Lubin.pnt-converted.txt
? UMP-Wroclaw/src/POI-Olawa-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Olesnica.pnt-converted.txt
? UMP-Wroclaw/src/POI-Olesnica.routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Swidnica.pnt-converted.txt
? UMP-Wroclaw/src/POI-Trzebnica-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Trzebnica.pnt-converted.txt
? UMP-Wroclaw/src/POI-Walbrzych-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Walbrzych.pnt-converted.txt
? UMP-Wroclaw/src/POI-Wroclaw-routing.pnt-converted.txt
? UMP-Wroclaw/src/POI-Wroclaw.inne.pnt-converted.txt
? UMP-Wroclaw/src/POI-Wroclaw.pnt-converted.txt
? UMP-Wroclaw/src/cities-Olesnica.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.bankomaty.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.budynki.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.geografia.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.inne.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.kolej.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.koscioly.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.paliwo.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.routing.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.szkoly.pnt-converted.txt
? UMP-Zamosc/src/POI-Zamosc.topo.pnt-converted.txt
? UMP-Zamosc/src/cities-Zamosc.pnt-converted.txt
? UMP-szlaki/src/CIECHANOW.szlaki.pnt-converted.txt
? UMP-szlaki/src/GDANSK.szlaki.pnt-converted.txt
? UMP-szlaki/src/POI-Poznan.szlaki.topo.pnt-converted.txt
? UMP-szlaki/src/POZNAN.szlaki.pnt-converted.txt
? UMP-szlaki/src/SUWALKI.szlaki.pnt-converted.txt
? UMP-szlaki/src/WARSZAWA.szlaki.pnt-converted.txt
Index: UMP-Bialystok/src/BIALYSTOK.ulice.txt
===================================================================
RCS file: /home/cvsroot/UMP-Bialystok/src/BIALYSTOK.ulice.txt,v
retrieving revision 1.82
diff -u -r1.82 BIALYSTOK.ulice.txt
--- UMP-Bialystok/src/BIALYSTOK.ulice.txt	17 May 2009 20:31:37 -0000	1.82
+++ UMP-Bialystok/src/BIALYSTOK.ulice.txt	10 Jun 2009 11:14:22 -0000
@@ -7064,7 +7064,7 @@
 Label=~[0x2e]65 Baranowicka
 EndLevel=4
 Data0=(53.12207,23.20314),(53.12206,23.20331),(53.12213,23.20561),(53.12218,23.20641),(53.12224,23.20731),(53.12241,23.20961),(53.12248,23.21053),(53.12265,23.21285),(53.12284,23.21540),(53.12295,23.21688),(53.12302,23.21789),(53.12338,23.22194),(53.12360,23.22443),(53.12367,23.22538),(53.12393,23.22670),(53.12430,23.22780),(53.12451,23.22833),(53.12516,23.22986)
-Label2-=Baranowicka
+Label2=Baranowicka
 [END]
 
 [POLYLINE]
Index: UMP-Dania/src/DANIA.obszary.txt
===================================================================
RCS file: /home/cvsroot/UMP-Dania/src/DANIA.obszary.txt,v
retrieving revision 1.22
diff -u -r1.22 DANIA.obszary.txt
--- UMP-Dania/src/DANIA.obszary.txt	5 Jun 2009 23:40:43 -0000	1.22
+++ UMP-Dania/src/DANIA.obszary.txt	10 Jun 2009 11:14:32 -0000
@@ -284,11 +284,11 @@
 Data0=(55.13700,14.11810),(55.14047,14.00000)
 [END]
 
-[POLYLINE]
-Type=0x1e
-Label=TSS Bornholmsgat
-EndLevel=3
-[END]
+;[POLYLINE]
+;Type=0x1e
+;Label=TSS Bornholmsgat
+;EndLevel=3
+;[END]
 
 [POLYLINE]
 Type=0x1d
Index: UMP-Hiszpania/src/Hiszpania-miasta_poza_obszarem_mapy.pnt.txt
===================================================================
RCS file: /home/cvsroot/UMP-Hiszpania/src/Hiszpania-miasta_poza_obszarem_mapy.pnt.txt,v
retrieving revision 1.2
diff -u -r1.2 Hiszpania-miasta_poza_obszarem_mapy.pnt.txt
--- UMP-Hiszpania/src/Hiszpania-miasta_poza_obszarem_mapy.pnt.txt	7 Apr 2009 08:46:21 -0000	1.2
+++ UMP-Hiszpania/src/Hiszpania-miasta_poza_obszarem_mapy.pnt.txt	10 Jun 2009 11:14:32 -0000
@@ -1,3 +1,11 @@
+;Afryka
+OziExplorer Point File Version 1.0
+WGS 84
+Reserved 1
+Reserved 2
+255,65535,3,8,0,0,CITY Hiszpania
+  35.91000,  -5.30000,  6,Ceuta,,,
+  35.30000,  -2.95000,  6,Melilla
 ;Canary Islands
   28.10000,  -15.43000,  8,Las Palmas
   28.47000,  -16.25000,  8,Santa Cruz de Tenerife
Index: UMP-JeleniaGora/src/miasta.txt
===================================================================
RCS file: /home/cvsroot/UMP-JeleniaGora/src/miasta.txt,v
retrieving revision 1.3
diff -u -r1.3 miasta.txt
--- UMP-JeleniaGora/src/miasta.txt	12 Feb 2007 02:20:03 -0000	1.3
+++ UMP-JeleniaGora/src/miasta.txt	10 Jun 2009 11:14:32 -0000
@@ -1,90 +0,0 @@
-[Countries]
-Country1=Polska~[0x1d]PL
-[END-Countries]
-
-[Regions]
-Region1=dolno¶l±skie
-CountryIdx1=1
-[END-Regions]
-
-[Cities]
-City1=Szklarska Porêba
-RegionIdx1=1
-City2=Jelenia Góra
-RegionIdx2=1
-City3=Zgorzelec
-RegionIdx3=1
-City4=Krzywa
-RegionIdx4=1
-City5=Boles³awiec
-RegionIdx5=1
-City6=kamienna góra
-RegionIdx6=1
-City7=kowary
-RegionIdx7=1
-City8=cieplice
-RegionIdx8=1
-City9=lwówek ¶l.
-RegionIdx9=1
-City10=gryfów ¶l.
-RegionIdx10=1
-City11=b³a¿ejów
-RegionIdx11=1
-City12=che³msko ¶l.
-RegionIdx12=1
-City13=GOLIñSK
-RegionIdx13=1
-City14=MIEROSZóW
-RegionIdx14=1
-City15=nowe siod³o
-RegionIdx15=1
-City16=okrzeszyn
-RegionIdx16=1
-City17=uniemy¶l
-RegionIdx17=1
-City18=Lubiatów
-RegionIdx18=1
-City19=Nowa sól
-RegionIdx19=1
-City20=olszyna
-RegionIdx20=1
-City21=wêgliniec
-RegionIdx21=1
-City22=lubañ
-RegionIdx22=1
-City23=góra
-RegionIdx23=1
-City24=serby
-RegionIdx24=1
-City25=radwanice
-RegionIdx25=1
-City26=lubsko
-RegionIdx26=1
-City27=G³ogów
-RegionIdx27=1
-City28=¶wiêtno
-RegionIdx28=1
-City29=mochy
-RegionIdx29=1
-City30=kaszczor
-RegionIdx30=1
-City31=wijewo
-RegionIdx31=1
-City32=Lipno
-RegionIdx32=1
-City33=Smigiel
-RegionIdx33=1
-City34=Wschowa
-RegionIdx34=1
-City35=Slawa
-RegionIdx35=1
-City36=szlichtyngowa
-RegionIdx36=1
-City37=konotop
-RegionIdx37=1
-City38=otyn
-RegionIdx38=1
-City39=kozuchow
-RegionIdx39=1
-[END-Cities]
-
Index: UMP-Kalisz/src/POI-Kalisz.pnt
===================================================================
RCS file: /home/cvsroot/UMP-Kalisz/src/POI-Kalisz.pnt,v
retrieving revision 1.86
diff -u -r1.86 POI-Kalisz.pnt
--- UMP-Kalisz/src/POI-Kalisz.pnt	12 Mar 2009 01:04:56 -0000	1.86
+++ UMP-Kalisz/src/POI-Kalisz.pnt	10 Jun 2009 11:14:32 -0000
@@ -15,8 +15,6 @@
   51.60072,  18.77262,  0,Pomnik Bohaterom wrzesnia 1939,,Sieradz,POMNIK
   51.59860,  18.69875,  0,Elektrocieplownia,Zachodnia,Sieradz,TOWER
   51.82535,  17.94680,  0,Kamieñ Œw. Jadwigi,,,ATRAKCJA
-  51.45717,  18.93333,  0,,,,0x1809
-  51.45275,  18.93333,  0,,,,0x1809
   51.19832,  18.85636,  0,Mokry Las,,,REZERWAT
   51.59197,  18.74720,  0,Most,,Sieradz,MOST
   51.60008,  18.76931,  0,Most,,Sieradz,MOST
Index: UMP-NowySacz/src/POI-NowySacz1.pnt
===================================================================
RCS file: /home/cvsroot/UMP-NowySacz/src/POI-NowySacz1.pnt,v
retrieving revision 1.98
diff -u -r1.98 POI-NowySacz1.pnt
--- UMP-NowySacz/src/POI-NowySacz1.pnt	1 Jun 2009 07:54:33 -0000	1.98
+++ UMP-NowySacz/src/POI-NowySacz1.pnt	10 Jun 2009 11:14:34 -0000
@@ -1225,7 +1225,7 @@
   49.61331,  20.69868,  0,,,Nowy S¹cz,0x1616
   49.60945,  20.70065,  0,,,Nowy S¹cz,0x1616
   49.61735,  20.71489,  0,,,Nowy S¹cz,0x1616
-  49.60800,  20.69426,  0,,,Nowy S¹cz,0x1916
+  49.60800,  20.69426,  0,,,Nowy S¹cz,0x1616
   49.60829,  20.69228,  0,,,Nowy S¹cz,0x1616
   49.60811,  20.69523,  0,,,Nowy S¹cz,0x1616
   49.62421,  20.71926,  0,,,Nowy S¹cz,0x1616
@@ -2090,9 +2090,9 @@
   49.42168,  20.95914,  0,Kredyt Bank,Zdrojowa;35;+48184716487,Krynica-Zdrój,ATM
   49.41110,  20.95534,  0,BPS ATM,Kraszewskiego;37;+48184715573,Krynica-Zdrój,ATM
   49.41229,  20.95540,  0,U Edyty,,Krynica-Zdrój,RESTAURACJA
-  49.63303,  20.71183,  0,,,Nowy S¹cz,0x1716
-  49.62076,  20.71825,  0,,,Nowy S¹cz,0x1716
-  49.61978,  20.71739,  0,,,Nowy S¹cz,0x1716
+  49.63303,  20.71183,  0,,,Nowy S¹cz,0x1616
+  49.62076,  20.71825,  0,,,Nowy S¹cz,0x1616
+  49.61978,  20.71739,  0,,,Nowy S¹cz,0x1616
   49.63542,  20.69216,  0,Media Markt,Tarnowska,Nowy S¹cz,RTV
   49.58897,  20.59580,  0,Grodzisko,,,ATRAKCJA
   49.58528,  20.60697,  0,Cmentarz Ewangelicki,,,CMENTARZ
Index: UMP-Opole/src/OPOLE.szlaki.topo.txt
===================================================================
RCS file: /home/cvsroot/UMP-Opole/src/OPOLE.szlaki.topo.txt,v
retrieving revision 1.39
diff -u -r1.39 OPOLE.szlaki.topo.txt
--- UMP-Opole/src/OPOLE.szlaki.topo.txt	30 May 2009 20:20:05 -0000	1.39
+++ UMP-Opole/src/OPOLE.szlaki.topo.txt	10 Jun 2009 11:14:35 -0000
@@ -571,8 +571,7 @@
 Label=Zielony / Niebieski
 EndLevel=1
 Data0=(50.42979,18.96325),(50.42924,18.96325),(50.42774,18.96303),(50.42611,18.96281),(50.42515,18.96266),(50.42328,18.96306),(50.42274,18.96315),(50.42057,18.96306),(50.41862,18.96299),(50.41638,18.96289),(50.41588,18.96260)
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
-Label2=Szlak Œwierklaniecki 
+Label2=Szlak Tysi¹clecia Pañstwa Polskiego / Szlak Œwierklaniecki 
 [END]
 
 [POLYLINE]
@@ -604,8 +603,7 @@
 Label=Zielony / Czerwony
 EndLevel=1
 Data0=(50.41221,18.94011),(50.41181,18.93965),(50.41109,18.93926),(50.41022,18.93892),(50.40950,18.93863),(50.40905,18.93845),(50.40878,18.94025),(50.40844,18.94218),(50.40816,18.94390),(50.40805,18.94416),(50.40659,18.94355),(50.40617,18.94343),(50.40600,18.94461),(50.40504,18.94398),(50.40357,18.94315),(50.40266,18.94261),(50.40172,18.94208),(50.40126,18.94189),(50.40027,18.94177),(50.39784,18.94168),(50.39498,18.94157),(50.39315,18.94151),(50.39169,18.94159)
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
-Label2=Szlak Husarii Polskiej
+Label2=Szlak Tysi¹clecia Pañstwa Polskiego / Szlak Husarii Polskiej
 [END]
 
 [POLYLINE]
@@ -637,8 +635,7 @@
 Label=Niebieski / Czerwony
 EndLevel=1
 Data0=(50.37823,18.93955),(50.37801,18.93954),(50.37731,18.93942),(50.37615,18.93919),(50.37491,18.93899),(50.37399,18.93883),(50.37395,18.93937),(50.37388,18.94022),(50.37382,18.94119),(50.37380,18.94188),(50.37380,18.94234),(50.37381,18.94273),(50.37373,18.94287),(50.37335,18.94261),(50.37304,18.94181),(50.37248,18.94035),(50.37206,18.93928),(50.37151,18.93792),(50.37149,18.93734),(50.37118,18.93618),(50.37078,18.93576),(50.37069,18.93512),(50.37060,18.93372)
-Label2=Szlak Powstañców Œl¹skich
-Label2=Szlak Husarii Polskiej
+Label2=Szlak Powstañców Œl¹skich / Szlak Husarii Polskiej
 [END]
 
 [POLYLINE]
@@ -646,8 +643,7 @@
 Label=Niebieski / Czerwony
 EndLevel=1
 Data0=(50.36725,18.93041),(50.36718,18.92998),(50.36710,18.92981),(50.36639,18.92922),(50.36585,18.92872),(50.36593,18.92857),(50.36589,18.92843),(50.36582,18.92835),(50.36571,18.92858),(50.36556,18.92841),(50.36549,18.92822),(50.36551,18.92805),(50.36604,18.92697),(50.36638,18.92633),(50.36668,18.92583),(50.36677,18.92563),(50.36668,18.92543),(50.36654,18.92535),(50.36570,18.92533),(50.36494,18.92534),(50.36440,18.92539),(50.36397,18.92530),(50.36355,18.92523),(50.36258,18.92461),(50.36160,18.92408),(50.36145,18.92378),(50.36055,18.92319),(50.35939,18.92254),(50.35875,18.92219),(50.35807,18.92203),(50.35698,18.92194),(50.35645,18.92177),(50.35528,18.92134),(50.35447,18.92102),(50.35280,18.92011),(50.35176,18.91958),(50.35034,18.91898),(50.34943,18.91858),(50.34826,18.91823)
-Label2=Szlak Powstañców Œl¹skich
-Label2=Szlak Husarii Polskiej
+Label2=Szlak Powstañców Œl¹skich / Szlak Husarii Polskiej
 [END]
 
 [POLYLINE]
@@ -655,8 +651,7 @@
 Label=Zielony / Niebieski
 EndLevel=1
 Data0=(50.34826,18.91823),(50.34806,18.91821),(50.34787,18.91814),(50.34758,18.91816),(50.34732,18.91832),(50.34714,18.91882),(50.34681,18.91869),(50.34607,18.91798),(50.34534,18.91732),(50.34476,18.91683),(50.34382,18.91599),(50.34364,18.91580)
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
-Label2=Szlak Powstañców Œl¹skich
+Label2=Szlak Tysi¹clecia Pañstwa Polskiego / Szlak Powstañców Œl¹skich
 [END]
 
 [POLYLINE]
@@ -680,8 +675,7 @@
 Label=Niebieski / Zielony
 EndLevel=1
 Data0=(50.39323,18.93205),(50.39382,18.92768),(50.39391,18.92660),(50.39431,18.92507),(50.39460,18.92427),(50.39486,18.92322),(50.39482,18.92228),(50.39512,18.92066),(50.39517,18.91994),(50.39574,18.91839),(50.39566,18.91770),(50.39624,18.91600),(50.39672,18.91430),(50.39739,18.91213),(50.39744,18.91174),(50.39736,18.91119),(50.39719,18.91079),(50.39735,18.90980),(50.39811,18.90792),(50.39843,18.90683),(50.39875,18.90526),(50.39897,18.90458),(50.39803,18.90331),(50.39737,18.90257),(50.39689,18.90222),(50.39667,18.90194),(50.39587,18.90137),(50.39565,18.90129),(50.39547,18.90130),(50.39536,18.90100),(50.39526,18.90085),(50.39518,18.90082),(50.39528,18.90023),(50.39534,18.90013),(50.39561,18.90000),(50.39562,18.89988),(50.39558,18.89970),(50.39540,18.89937),(50.39546,18.89915),(50.39561,18.89882)
-Label2=Szlak Powstañców Œl¹skich
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
+Label2=Szlak Powstañców Œl¹skich / Szlak Tysi¹clecia Pañstwa Polskiego
 [END]
 
 [POLYLINE]
@@ -705,8 +699,7 @@
 Label=Zielony / Niebieski
 EndLevel=1
 Data0=(50.39187,18.87401),(50.39189,18.87346),(50.39159,18.87204),(50.39124,18.87050),(50.39091,18.86955),(50.38979,18.86820),(50.38901,18.86725),(50.38872,18.86739),(50.38864,18.86716),(50.38851,18.86723),(50.38820,18.86703),(50.38789,18.86717),(50.38770,18.86721),(50.38736,18.86709),(50.38717,18.86730),(50.38695,18.86733),(50.38681,18.86722),(50.38654,18.86722)
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
-Label2=Szlak Powstañców Œl¹skich
+Label2=Szlak Tysi¹clecia Pañstwa Polskiego / Szlak Powstañców Œl¹skich
 [END]
 
 [POLYLINE]
@@ -730,8 +723,7 @@
 Label=¯ó³ty / Zielony
 EndLevel=1
 Data0=(50.37745,18.87738),(50.37709,18.87830),(50.37659,18.87959)
-Label2=Szlak im. Z. Kleszczyñskiego
-Label2=Szlak Tysi¹clecia Pañstwa Polskiego
+Label2=Szlak im. Z. Kleszczyñskiego / Szlak Tysi¹clecia Pañstwa Polskiego
 [END]
 
 [POLYLINE]
@@ -803,8 +795,7 @@
 Label=Czerwony / ¯ó³ty
 EndLevel=1
 Data0=(50.34475,19.00143),(50.34472,19.00205),(50.34467,19.00278),(50.34460,19.00432),(50.34453,19.00556),(50.34448,19.00663),(50.34444,19.00744),(50.34429,19.01057),(50.34417,19.01332),(50.34412,19.01425),(50.34456,19.01444),(50.34551,19.01490),(50.34537,19.01573),(50.34509,19.01730),(50.34490,19.01811),(50.34450,19.01933),(50.34463,19.02151),(50.34485,19.02523),(50.34492,19.02621)
-Label2=Szlak Husarii Polskiej
-Label2=Szlak Historii Górnictwa Górnego Œl¹ska
+Label2=Szlak Husarii Polskiej / Szlak Historii Górnictwa Górnego Œl¹ska
 [END]
 
 [POLYLINE]
@@ -836,8 +827,7 @@
 Label=Niebieski / Czarny
 EndLevel=1
 Data0=(50.40796,18.84531),(50.40885,18.84479),(50.41011,18.84299),(50.41058,18.84249)
-Label2=Szlak Powstañców Œl¹skich
-Label2=Szlak Segiecki
+Label2=Szlak Powstañców Œl¹skich / Szlak Segiecki
 [END]
 
 [POLYLINE]
@@ -869,8 +859,7 @@
 Label=Niebieski / ¯ó³ty
 EndLevel=1
 Data0=(50.42730,18.80770),(50.42754,18.80664),(50.42877,18.80501),(50.42949,18.80351),(50.42998,18.80341),(50.43027,18.80315),(50.43020,18.80199),(50.43041,18.80077)
-Label2=Szlak Powstañców Œl¹skich
-Label2=Szlak Gwarków
+Label2=Szlak Powstañców Œl¹skich / Szlak Gwarków
 [END]
 
 [POLYLINE]
@@ -918,9 +907,7 @@
 Label=Czerwony / Czarny / ¯ó³ty
 EndLevel=1
 Data0=(50.44254,18.85251),(50.44283,18.85276),(50.44321,18.85295),(50.44350,18.85363),(50.44401,18.85386),(50.44396,18.85462),(50.44439,18.85483)
-Label2=Szlak Husarii Polskiej
-Label2=Szlak Segiecki
-Label2=Szlak Gwarków
+Label2=Szlak Husarii Polskiej / Szlak Segiecki / Szlak Gwarków
 [END]
 
 [POLYLINE]
@@ -936,8 +923,7 @@
 Label=Czerwony / ¯ó³ty
 EndLevel=1
 Data0=(50.44254,18.85251),(50.44242,18.85209),(50.44235,18.85150),(50.44191,18.85115),(50.44170,18.85053),(50.44105,18.84843),(50.44073,18.84737),(50.44051,18.84693)
-Label2=Szlak Husarii Polskiej
-Label2=Szlak Gwarków
+Label2=Szlak Husarii Polskiej / Szlak Gwarków
 [END]
 
 [POLYLINE]
@@ -961,8 +947,7 @@
 Label=Czarny / ¯ó³ty
 EndLevel=1
 Data0=(50.44439,18.85483),(50.44486,18.85507),(50.44498,18.85489),(50.44548,18.85523),(50.44615,18.85544),(50.44613,18.85676),(50.44601,18.85796),(50.44586,18.85900),(50.44573,18.85998),(50.44588,18.86016),(50.44594,18.86112),(50.44630,18.86107),(50.44630,18.86125),(50.44600,18.86130),(50.44587,18.86141),(50.44607,18.86191),(50.44675,18.86252),(50.44720,18.86356),(50.44745,18.86404)
-Label2=Szlak Segiecki
-Label2=Szlak Gwarków
+Label2=Szlak Segiecki / Szlak Gwarków
 [END]
 
 [POLYLINE]
@@ -970,8 +955,7 @@
 Label=Czarny / ¯ó³ty / Zielony
 EndLevel=1
 Data0=(50.44745,18.86404),(50.44847,18.86476)
-Label2=Szlak Segiecki
-Label2=Szlak Gwarków
+Label2=Szlak Segiecki / Szlak Gwarków
 [END]
 
 [POLYLINE]
@@ -1050,8 +1034,7 @@
 Label=¯ó³ty / Czarny / Zielony
 EndLevel=1
 Data0=(50.46419,18.90383),(50.46410,18.90502),(50.46396,18.90689),(50.46389,18.90803)
-Label2=Szlak Gwarków
-Label2=Szlak Nakielski
+Label2=Szlak Gwarków / Szlak Nakielski
 [END]
 
 [POLYLINE]
Index: UMP-Portugalia/src/Portugalia-miasta_poza_obszarem_mapy.pnt.txt
===================================================================
RCS file: /home/cvsroot/UMP-Portugalia/src/Portugalia-miasta_poza_obszarem_mapy.pnt.txt,v
retrieving revision 1.1
diff -u -r1.1 Portugalia-miasta_poza_obszarem_mapy.pnt.txt
--- UMP-Portugalia/src/Portugalia-miasta_poza_obszarem_mapy.pnt.txt	8 May 2008 23:04:16 -0000	1.1
+++ UMP-Portugalia/src/Portugalia-miasta_poza_obszarem_mapy.pnt.txt	10 Jun 2009 11:14:35 -0000
@@ -1,8 +1,13 @@
 ;Madeira
+;Açores
+OziExplorer Point File Version 1.0
+WGS 84
+Reserved 1
+Reserved 2
+255,65535,3,8,0,0,CITY Portugalia
   32.65000,  -16.90000,  6,Funchal
   32.63000,  -16.96000,  5,Câmara de Lobos
   32.63000,  -16.84000,  4,Caniço
   32.70000,  -16.76000,  4,Machico
-;Açores
   37.75000,  -25.67000,  5,Ponta Delgada
   38.66000,  -27.22000,  4,Angra de Heroismo
Index: UMP-Poznan/src/POZNAN.ulice.txt
===================================================================
RCS file: /home/cvsroot/UMP-Poznan/src/POZNAN.ulice.txt,v
retrieving revision 1.848
diff -u -r1.848 POZNAN.ulice.txt
--- UMP-Poznan/src/POZNAN.ulice.txt	3 Jun 2009 14:19:56 -0000	1.848
+++ UMP-Poznan/src/POZNAN.ulice.txt	10 Jun 2009 11:14:41 -0000
@@ -8195,7 +8195,6 @@
 DirIndicator=1
 Data0=(52.38499,16.97139),(52.38419,16.97231)
 Label2=Boles³awa Krzywoustego
-Label2=Boles³awa Krzywoustego
 [END]
 
 [POLYLINE]
@@ -31833,8 +31832,8 @@
 Label=Wergiliusza
 EndLevel=1
 Data0=(52.44443,16.88162),(52.44455,16.88242)
--Numbers1=0,O,45,47,N,-1,-1
--Numbers2=1,N,-1,-1,N,-1,-1
+Numbers1=0,O,45,47,N,-1,-1
+Numbers2=1,N,-1,-1,N,-1,-1
 [END]
 
 [POLYLINE]
Index: UMP-Przemysl/src/Pruchnik-pliki.txt
===================================================================
RCS file: /home/cvsroot/UMP-Przemysl/src/Pruchnik-pliki.txt,v
retrieving revision 1.3
diff -u -r1.3 Pruchnik-pliki.txt
--- UMP-Przemysl/src/Pruchnik-pliki.txt	13 Mar 2007 22:38:11 -0000	1.3
+++ UMP-Przemysl/src/Pruchnik-pliki.txt	10 Jun 2009 11:14:41 -0000
@@ -1,8 +0,0 @@
-PRUCHNIK.obszary.txt
-PRUCHNIK.szlaki.txt
-PRUCHNIK.ulice.txt
-PRUCHNIK.woda.txt
-PRUCHNIK.zielone.txt
-PRUCHNIK.warstwice.txt
-PRUCHNIK.drogi.txt
-POI-Pruchnik.txt
Index: UMP-Szczecin/src/STARGARD_SZCZECINSKI.ulice.txt
===================================================================
RCS file: /home/cvsroot/UMP-Szczecin/src/STARGARD_SZCZECINSKI.ulice.txt,v
retrieving revision 1.45
diff -u -r1.45 STARGARD_SZCZECINSKI.ulice.txt
--- UMP-Szczecin/src/STARGARD_SZCZECINSKI.ulice.txt	16 May 2009 08:44:28 -0000	1.45
+++ UMP-Szczecin/src/STARGARD_SZCZECINSKI.ulice.txt	10 Jun 2009 11:14:43 -0000
@@ -560,7 +560,6 @@
 Label=plac S³oneczny
 EndLevel=1
 Data0=(53.33828,15.02069),(53.33815,15.01977)
-End Level=1
 [END]
 
 [POLYLINE]
Index: UMP-Tarnow/src/TARNOW.woda.txt
===================================================================
RCS file: /home/cvsroot/UMP-Tarnow/src/TARNOW.woda.txt,v
retrieving revision 1.83
diff -u -r1.83 TARNOW.woda.txt
--- UMP-Tarnow/src/TARNOW.woda.txt	30 Apr 2009 08:09:51 -0000	1.83
+++ UMP-Tarnow/src/TARNOW.woda.txt	10 Jun 2009 11:14:46 -0000
@@ -1357,4 +1357,3 @@
 EndLevel=1
 Data0=(50.00951,20.76430),(50.00885,20.76411),(50.00826,20.76348),(50.00751,20.76372),(50.00676,20.76307),(50.00654,20.76162),(50.00602,20.76059),(50.00591,20.75987),(50.00566,20.75959),(50.00517,20.75970),(50.00439,20.75950),(50.00370,20.75896),(50.00358,20.75835),(50.00326,20.75828),(50.00276,20.75875),(50.00148,20.75847),(50.00139,20.75802),(50.00104,20.75775),(50.00093,20.75724),(50.00075,20.75717),(50.00022,20.75742),(49.99999,20.75726),(49.99970,20.75772),(49.99889,20.75739),(49.99852,20.75719),(49.99812,20.75716),(49.99749,20.75675),(49.99654,20.75698),(49.99603,20.75665),(49.99559,20.75662),(49.99512,20.75737),(49.99420,20.75702),(49.99398,20.75762),(49.99396,20.75763),(49.99359,20.75773),(49.99350,20.75812),(49.99304,20.75838),(49.99279,20.75797),(49.99240,20.75815),(49.99164,20.75834),(49.99099,20.75931),(49.99028,20.75886),(49.98990,20.75882),(49.98985,20.75946),(49.98923,20.75926),(49.98859,20.75998),(49.98776,20.75960),(49.98736,20.75976),(49.98711,20.76066),(49.98649,20.76122),(49.98585,20.76210),(49.98447,20.76300),(49.98277,20.76361),(49.98161,20.76395),(49.98053,20.76326),(49.97975,20.76133),(49.97667,20.75662),(49.97630,20.75527),(49.97591,20.75530),(49.97547,20.75500),(49.97433,20.75604),(49.97391,20.75600),(49.97386,20.75552),(49.97372,20.75515),(49.97331,20.75518),(49.97305,20.75473),(49.97283,20.75457),(49.97224,20.75474),(49.97188,20.75457),(49.97004,20.75466),(49.96963,20.75444),(49.96927,20.75467),(49.96901,20.75452),(49.96871,20.75464),(49.96836,20.75445),(49.96674,20.75483),(49.96631,20.75586),(49.96600,20.75610),(49.96519,20.75567),(49.96508,20.75482),(49.96483,20.75456),(49.96451,20.75352),(49.96294,20.75228)
 [END]
-
Index: UMP-Ukraina/src/miasta_poza_obszarem_mapy.pnt.txt
===================================================================
RCS file: /home/cvsroot/UMP-Ukraina/src/miasta_poza_obszarem_mapy.pnt.txt,v
retrieving revision 1.1
diff -u -r1.1 miasta_poza_obszarem_mapy.pnt.txt
--- UMP-Ukraina/src/miasta_poza_obszarem_mapy.pnt.txt	8 May 2008 20:41:24 -0000	1.1
+++ UMP-Ukraina/src/miasta_poza_obszarem_mapy.pnt.txt	10 Jun 2009 11:14:47 -0000
@@ -1,3 +1,8 @@
+OziExplorer Point File Version 1.0
+WGS 84
+Reserved 1
+Reserved 2
+255,65535,3,8,0,0,CITY Ukraina
   49.98000,  36.22000, 10,Charkiv
   48.45000,  34.98000, 10,Dnipropetrovsk
   48.00000,  37.82000, 10,Doneck
Index: UMP-Wroclaw/src/LEGNICA.ulice.txt
===================================================================
RCS file: /home/cvsroot/UMP-Wroclaw/src/LEGNICA.ulice.txt,v
retrieving revision 1.18
diff -u -r1.18 LEGNICA.ulice.txt
--- UMP-Wroclaw/src/LEGNICA.ulice.txt	9 Apr 2009 09:45:25 -0000	1.18
+++ UMP-Wroclaw/src/LEGNICA.ulice.txt	10 Jun 2009 11:14:47 -0000
@@ -3449,7 +3449,6 @@
 EndLevel=1
 Data0=(51.199161,16.146429),(51.199001,16.148126)
 Label2=Puchatka (Kubusia)
-Label2=Puchatka
 [END]
 
 [POLYLINE]
Index: UMP-Wroclaw/src/WALBRZYCH.drogi.txt
===================================================================
RCS file: /home/cvsroot/UMP-Wroclaw/src/WALBRZYCH.drogi.txt,v
retrieving revision 1.242
diff -u -r1.242 WALBRZYCH.drogi.txt
--- UMP-Wroclaw/src/WALBRZYCH.drogi.txt	4 Jun 2009 11:10:14 -0000	1.242
+++ UMP-Wroclaw/src/WALBRZYCH.drogi.txt	10 Jun 2009 11:14:51 -0000
@@ -19411,7 +19411,7 @@
 Label=Solna
 EndLevel=1
 Data0=(50.849014,16.285631),(50.848915,16.285217),(50.848781,16.284832),(50.848636,16.284431),(50.848556,16.283924),(50.848533,16.283473),(50.848514,16.283012),(50.848548,16.282102),(50.848716,16.281440),(50.848953,16.280878)
-Misto=Œwiebodzice
+Miasto=Œwiebodzice
 [END]
 
 [POLYLINE]
Index: UMP-Wroclaw/src/WROCLAW.drogi.txt
===================================================================
RCS file: /home/cvsroot/UMP-Wroclaw/src/WROCLAW.drogi.txt,v
retrieving revision 1.295
diff -u -r1.295 WROCLAW.drogi.txt
--- UMP-Wroclaw/src/WROCLAW.drogi.txt	6 Jun 2009 18:15:49 -0000	1.295
+++ UMP-Wroclaw/src/WROCLAW.drogi.txt	10 Jun 2009 11:14:52 -0000
@@ -4815,7 +4815,7 @@
 
 [POLYLINE]
 Type=0x3
-Label=~[0c2c]8
+Label=~[0x2c]8
 EndLevel=2
 DirIndicator=1
 Data0=(51.041831,16.969509),(51.042610,16.969677),(51.042919,16.969734),(51.043502,16.969905),(51.044017,16.970125),(51.044712,16.970420),(51.045047,16.970563),(51.046463,16.970607),(51.047050,16.970624),(51.047328,16.970684),(51.047584,16.970848)
Index: UMP-Wroclaw/src/WROCLAW.obszary.txt
===================================================================
RCS file: /home/cvsroot/UMP-Wroclaw/src/WROCLAW.obszary.txt,v
retrieving revision 1.52
diff -u -r1.52 WROCLAW.obszary.txt
--- UMP-Wroclaw/src/WROCLAW.obszary.txt	27 Apr 2009 22:13:40 -0000	1.52
+++ UMP-Wroclaw/src/WROCLAW.obszary.txt	10 Jun 2009 11:14:52 -0000
@@ -1041,6 +1041,6 @@
 Label=Stacja trafo
 EndLevel=1
 Data0=(51.034487,17.096501),(51.034191,17.097755),(51.032847,17.096738),(51.033180,17.095551)
-Miast=Zacharzyce
+Miasto=Zacharzyce
 [END]
 
Index: narzedzia/pnt2poi.awk
===================================================================
RCS file: /home/cvsroot/narzedzia/pnt2poi.awk,v
retrieving revision 1.63
diff -u -r1.63 pnt2poi.awk
--- narzedzia/pnt2poi.awk	2 Jun 2009 12:16:02 -0000	1.63
+++ narzedzia/pnt2poi.awk	10 Jun 2009 11:14:52 -0000
@@ -22,7 +22,7 @@
 	sym["WERYFIKOWAC"]="0x170a";
 	sym["NIE_WIEM"]="0x170d";
 	sym["WATPLIWY"]="0x170d";
-	sym["ZAKAZ"]="0x1710";
+	sym["ZAKAZ"]="0x17105";
 	sym["REMONT"]="0x1712";
 
 	sym["STAWA"]="0x1606";
@@ -52,10 +52,10 @@
 	sym["JEDZENIE"]="0x2a00";
 	sym["AMERICAN"]="0x2a01";
 	sym["ASIAN"]="0x2a02";
-	sym["SUSHI"]="0x2a02";
-	sym["BARBECUE"]="0x2a03";
-	sym["GRILL"]="0x2a03";
-	sym["KEBAB"]="0x2a03";
+	sym["SUSHI"]="0x2a025";
+	sym["BARBECUE"]="0x2a030";
+	sym["GRILL"]="0x2a031";
+	sym["KEBAB"]="0x2a032";
 	sym["CHINESE"]="0x2a04";
 	sym["DELI"]="0x2a05";
 	sym["INTERNATIONA"]="0x2a06";
@@ -79,12 +79,12 @@
 	sym["GERMAN"]="0x2a10";
 	sym["BRITISH"]="0x2a11";
 	sym["GRECKA"]="0x2a12";
-	sym["LIBANSKA"]="0x2a12";
+	sym["LIBANSKA"]="0x2a125";
 
 	sym["HOSTEL"]="0x2b00";
 	sym["SCHRONISKO"]="0x2b00";	## ?? a moze 2b05?/2b04
 	sym["lodging"]="0x2b01";
-	sym["MOTEL"]="0x2b01";	# Hotel/Motel
+	sym["MOTEL"]="0x2b015";	# Hotel/Motel
 	sym["HOTEL"]="0x2b01";	# Hotel/Other
 	sym["B&B"]="0x2b02";
 	sym["camping"]="0x2b03";
@@ -99,15 +99,15 @@
 	sym["amusement_park"]="0x2c01";	# Amusement Park
 	sym["AMUSEMENT_PARK"]="0x2c01";
 	sym["PLAC_ZABAW"]="0x2c01";
-	sym["GALERIA"]="0x2c02";
+	sym["GALERIA"]="0x2c025";
 	sym["museum"]="0x2c02";
 	sym["muzea"]="0x2c02";
 	sym["MUZEUM"]="0x2c02";
 	sym["BIBLIOTEKA"]="0x2c03";
-	sym["DWOREK"]="0x2c04";
-	sym["PALAC"]="0x2c04";
-	sym["CASTLE"]="0x2c04";	# ???
-	sym["ZAMEK"]="0x2c04";	# ???
+	sym["DWOREK"]="0x2c040";
+	sym["PALAC"]="0x2c041";
+	sym["CASTLE"]="0x2c042";	# ???
+	sym["ZAMEK"]="0x2c043";	# ???
 	sym["school"]="0x2c05";
 	sym["SZKOLA"]="0x2c05";
 	sym["szkoly"]="0x2c05";
@@ -118,9 +118,9 @@
 	sym["PARK"]="0x2c06";
 	sym["zoo"]="0x2c07";
 	sym["ZOO"]="0x2c07";
-	sym["stadium"]="0x2c08";
-	sym["STADION"]="0x2c08";	# Sports(point)
-	sym["BOISKO"]="0x2c08";
+	sym["stadium"]="0x2c081";
+	sym["STADION"]="0x2c081";	# Sports(point)
+	sym["BOISKO"]="0x2c080";
 	sym["KONCERT"]="0x2c09";
 	sym["WINIARNIA"]="0x2c0a";
 	sym["WINERY"]="0x2c0a";
@@ -139,8 +139,8 @@
 	sym["KINO"]="0x2d03";
 	sym["MULTIKINO"]="0x2d03";
 	sym["kina"]="0x2d03";	# Cinema
-	sym["CASINO"]="0x2d04";
-	sym["KASYNO"]="0x2d04";
+	sym["CASINO"]="0x2d045";
+	sym["KASYNO"]="0x2d045";
 	sym["GOLF"]="0x2d05";
 	sym["NARTY"]="0x2d06";	# Ski center
 	sym["KREGLE"]="0x2d07";
@@ -148,11 +148,11 @@
 	sym["BASEN"]="0x2d09";
 	sym["baseny"]="0x2d09";
 	sym["NUREK"]="0x2d09";
-	sym["fitness"]="0x2d0a";
-	sym["FITNESS"]="0x2d0a";	# Sports(point)
-	sym["KORT"]="0x2d0a";	# Sports(point)
-	sym["KORTY"]="0x2d0a";	# Sports(point)
-	sym["SKATEPARK"]="0x2d0a";	# Sports(point)
+	sym["fitness"]="0x2d0a0";
+	sym["FITNESS"]="0x2d0a0";	# Sports(point)
+	sym["KORT"]="0x2d0a1";	# Sports(point)
+	sym["KORTY"]="0x2d0a1";	# Sports(point)
+	sym["SKATEPARK"]="0x2d0a2";	# Sports(point)
 	sym["SPORT"]="0x2d0a";	# Sports(point)
 	sym["landing"]="0x2d0b";
 
@@ -168,7 +168,7 @@
 	sym["MARKET"]="0x2e01";
 	sym["store"]="0x2e01";	# Dept. Store
 	sym["SPOZYWCZY"]="0x2e02";
-	sym["BAZAR"]="0x2e02";
+	sym["BAZAR"]="0x2e025";
 	sym["GROCERY"]="0x2e02";
 	sym["SUPER"]="0x2e03";
 	sym["SUPERMARKET"]="0x2e03";
@@ -187,7 +187,7 @@
 	sym["DOM_I_OGROD"]="0x2e08";
 	sym["MEBLE"]="0x2e09";
 	sym["WNETRZA"]="0x2e09";
-	sym["ROWEROWY"]="0x2e0a";
+	sym["ROWEROWY"]="0x2e0a5";
 	sym["SPORTOWY"]="0x2e0a";
 	sym["TURYSTYCZNY"]="0x2e0a";
 	sym["RTV"]="0x2e0b";
@@ -213,23 +213,23 @@
 	sym["BANK"]="0x2f06";
 	sym["banki"]="0x2f06";
 	sym["CARDEALER"]="0x2f07";
-	sym["BUS"]="0x2f08";
-	sym["TRAM"]="0x2f08";
-	sym["METRO"]="0x2f08";
-	sym["PKS"]="0x2f08";
-	sym["PKP"]="0x2f08";	# bus/pkp
+	sym["BUS"]="0x2f080";
+	sym["TRAM"]="0x2f081";
+	sym["METRO"]="0x2f082";
+	sym["PKS"]="0x2f083";
+	sym["PKP"]="0x2f084";	# bus/pkp
 	sym["PORT"]="0x2f09";
 	sym["PRZYSTAN"]="0x2f09";
 	sym["parking"]="0x2f0b";	# 4d00?
 	sym["PARKING"]="0x2f0b";	# 4d00?
-	sym["info"]="0x2f0c";
-	sym["INFO"]="0x2f0c";
-	sym["INFORMACJA"]="0x2f0c";
+	sym["info"]="0x2f0c5";
+	sym["INFO"]="0x2f0c5";
+	sym["INFORMACJA"]="0x2f0c5";
 	sym["RESTROOM"]="0x2f0c";	# ???
 	sym["CARWASH"]="0x2f0e";
 	sym["MYJNIA"]="0x2f0e";
 	sym["GARMIN"]="0x2f0f";
-	sym["TATOO"]="0x2f10";	# personal services
+	sym["TATOO"]="0x2f105";	# personal services
 	sym["FABRYKA"]="0x2f11";
 	sym["BUSINESS"]="0x2f11";
 	sym["FIRMA"]="0x2f11";
@@ -242,13 +242,13 @@
 
 	sym["EMERGENCY"]="0x3000";	
 	sym["POLICJA"]="0x3001";
-	sym["DENTYSTA"]="0x3002";
+	sym["DENTYSTA"]="0x30027";
 	sym["POGOTOWIE"]="0x3002";
-	sym["PRZYCHODNIA"]="0x3002";
+	sym["PRZYCHODNIA"]="0x30025";
 	sym["SZPITAL"]="0x3002";
 	sym["szpitale"]="0x3002";
-	sym["UZDROWISKO"]="0x3002";
-	sym["WETERYNARZ"]="0x3002";
+	sym["UZDROWISKO"]="0x30025";
+	sym["WETERYNARZ"]="0x30026";
 	sym["RATUSZ"]="0x3003";
 	sym["SAD"]="0x3004";
 	sym["toll"]="0x3006";
@@ -271,11 +271,11 @@
 	sym["skiing"]="0x5300";
 	sym["SKI"]="0x5300";
 	sym["KAPIELISKO"]="0x5400";
-	sym["FA"]="0x5600"; # Atrapa fotoradaru
-	sym["FP"]="0x5600"; # Fotoradar przenosny
-	sym["FS"]="0x5600"; # Fotoradar staly
-	sym["KD"]="0x5600"; # Kontrola drogowa
-	sym["RA"]="0x5600"; # Radiowoz z kamera
+	sym["FA"]="0x56005"; # Atrapa fotoradaru
+	sym["FP"]="0x56005"; # Fotoradar przenosny
+	sym["FS"]="0x56005"; # Fotoradar staly
+	sym["KD"]="0x56005"; # Kontrola drogowa
+	sym["RA"]="0x56005"; # Radiowoz z kamera
 	sym["CZARNYPUNKT"]="0x5700";
 	sym["DANGER"]="0x5700";
 	sym["FOTORADAR"]="0x5700";
@@ -311,15 +311,15 @@
 	sym["PLATFORMA"]="0x640d";
 	sym["RV_park"]="0x640e";
 	sym["REZERWAT"]="0x640e";	# park
-	sym["short_tower"]="0x6411";
-	sym["tall_tower"]="0x6411";
+	sym["short_tower"]="0x64110";
+	sym["tall_tower"]="0x64111";
 	sym["TOWER"]="0x6411";
 	sym["WIEZA"]="0x6411";
 	sym["TRAIL"]="0x6412";	# ???
 	sym["SZLAK"]="0x6412";
 	sym["TUNEL"]="0x6413";
-	sym["JASKINIA"]="0x6413";
-	sym["CAVE"]="0x6413";	# Tunel?
+	sym["JASKINIA"]="0x64135";
+	sym["CAVE"]="0x64135";	# Tunel?
 	sym["OLIGOCEN"]="0x6414";
 	sym["BUNKIER"]="0x6415";	# ???
 	sym["FORT"]="0x6415";	# ???
@@ -331,8 +331,8 @@
 	sym["boat_ramp"]="0x650b"; # Harbor
 	sym["WYSPA"]="0x650c";
 	sym["JEZIORO"]="0x650d";
-	sym["TOITOI"]="0x650f";
-	sym["WC"]="0x650f";	# ???
+	sym["TOITOI"]="0x650f5";
+	sym["WC"]="0x650f5";	# ???
 	sym["stream"]="0x6512";
 	sym["pond"]="0x6513";
 	sym["KURHAN"]="0x6600";	# ???
@@ -347,7 +347,7 @@
 	sym["TREE"]="0x660a";
 	sym["elevation"]="0x6610";
 	sym["plain"]="0x6610";
-	sym["short_tower"]="0x6611";
+	sym["short_tower"]="0x64110";
 	sym["SKALA"]="0x6614";
 	sym["GORA"]="0x6616";
 	sym["HILL"]="0x6616";
